### DISCUSSION TITLE
Can ab-initio calculation interface with Q-chem be added?

### DISCUSSION THREAD

**Discussion by Cynthia-0807** [deepmodeling/dpgen#656] [Q&A]
I have been using Q-chem to calculation the single point energy.However,dpgen doesn't support the interface with Q-chem.The location of the file about QM calculation is dpgen/generator/lib? I am new about python,so I don't understand those files in aforementioned folder..........

**Comment by AnguseZhang** [comment]
We're not familiar with Q-Chem, so we can't implement DP-GEN's interface with Q-Chem. If someone wants to implement this function, we would like to support.

**Reply by Cynthia-0807** [reply]
Thank for your reply!I think the Q-chem software is user-friendly,and is a little similar to Gaussian.
